spark/src/test/scala/org/apache/spark/sql/delta/stats/StatsCollectionSuite.scala

```diff
@@ -608,7 +590,8 @@ class StatsCollectionSuite
 
   Seq(
     "BIGINT", "DATE", "DECIMAL(3, 2)", "DOUBLE", "FLOAT", "INT", "SMALLINT", "STRING",
-    "TIMESTAMP", "TIMESTAMP_NTZ", "TINYINT"
+    "TIMESTAMP", "TIMESTAMP_NTZ", "TINYINT", "STRUCT<c3: BIGINT>",
```

Comment:
Looks like there are relevant tests in DataSkippingDeltaTests as well https://github.com/delta-io/delta/blob/da5a5d29b6f00265439bc84020580080a2d4568f/spark/src/test/scala/org/apache/spark/sql/delta/stats/DataSkippingDeltaTests.scala#L442

Reply:
Those tests are for specifying the number of indexed columns, not columns by name